### ISSUE TITLE
Create a vanilla development environment

### ISSUE THREAD

**Issue by celian-garcia** [prometheus-community/monaco-promql#10]
### Requirements
Would like to have a dev environment ultra easy to run, without any heavy framework to run like angular or react.

- **easy to run** : as easy as open an index.html file in a browser.
- **compiling fastly** : I'm tired of waiting for a life all the internet to be downloaded into this node_modules zorglub...
- **easy to understand** : the more easy to understand the less scary are the people to create PR

### More discussion
Being not an expert on how the web is really working (without everything hidden in a framework), it would be a very good opportunity to learn more about it.

Maybe we need at least a typescript -> javascript compilation with tsc command ? 
Maybe we can take this opportunity to make a bundled js file with bundle technology like webpack ?

**Comment by Nexucis** [comment]
Webpack is basically all you need normally. It should handle the typescript transpilation and also the web dev server. Same that you have in Angular

**Comment by Nexucis** [comment]
And fyi this dev vanilla env was something I was planning to do for codeMirror ;). In case you are wondering where does it Come from

**Comment by celian-garcia** [comment]
Yeah webpack seems a good candidate, I'm just planning to learn more on what is under the hood.

It has chances to end with webpack because of how it is easy to use. Hoping it's not too heavy to download let's see. 
Indeed webpack has this fancy webpack-dev-server component which permit to run a bundled js file easily.

At the end everything is possible, I assume that we can run a tsc command and a tiny web server as well. Let's see what is the easier and faster to run.

**Comment by Nexucis** [comment]
@celian-garcia I linked a PR from codemirror-promql that is doing that.
If you want I can do the same here

**Comment by celian-garcia** [comment]
Yeah I saw it.
I already know webpack so it's not fun, I want to use this issue to make a study and see what is existing else.
There is no urgence
